observatory_platform/tests/test_dataset_api.py

```diff
@@ -156,26 +156,26 @@ def test_get_latest_dataset_release(self):
                 api.add_dataset_release(release)
 
             latest = api.get_latest_dataset_release(dag_id=dag_id, entity_id=entity_id, date_key="snapshot_date")
-            self.assertEqual(releases[-1], latest)
+            self.assertEqual(releases[-1].to_dict(), latest.to_dict())
```

Comment:
The actual object should be compared (it is compared with the "__eq__" function), as the dict form is different.